cypress/integration/login.js

```diff
@@ -1,28 +1,111 @@
+import LoginPage from '../objects/loginPage';
+import TimeLogging from '../objects/timeLogging/timelogging'
+const loginPage = new LoginPage()
+
+
+
 describe('Sourcebooks login', function() {
 
     it('Should display validation for empty user after attempted loggin', function () {
-        
-        cy.visit('/');
-        cy.get('.Select.not-valid').should('not.visible')
-        cy.get('[type="submit"]').click();
-        cy.get('.Select.not-valid').should('be.visible')
+
+        loginPage.visit()
+        loginPage.getValidationError().should('not.visible')
+        loginPage.getLoginButton().click();
+        loginPage.getValidationError().should('be.visible')
     })
 
     it('Should be able to login with role User', function () {
 
+        loginPage.visit()
+        loginPage.getUser().click({force:true});
+        loginPage.getName('Laurynas Kochas').click();
+        loginPage.getRole().click({force:true});
+        loginPage.getRoleLabel('User').click();
+        loginPage.getLoginButton().click();
+
+        cy.url().should('include', '/time-logging');
+        cy.get('.page__title').contains('Timesheets');
+        cy.get('.calendar').should('be.visible');

```

Comment:
So everything's great, you've created some nice functions to retrieve the required elements from the Time Logging page, but then forgot to actually use them in your test, right? :)